src/ann_solo/reader.py

```diff
@@ -2,21 +2,26 @@
 import logging
 import os
 import pickle
+import gzip
```

Comment:
**nitpick:** Sort imports alphabetically. Put all `import ...` statements before the `from ... import ...` statements.